### PR TITLE
Add the multi-cards support for image_to_text examples

### PR DESCRIPTION
**Pull request by yuanwu2017**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by yafshar** [comment]
@yuanwu2017 can you update the README?

**Comment by yafshar** [comment]
@yuanwu2017 I am wondering why didn't you use `gaudi_spawn.py` which has everything you want to do here and be consistent with the rest of examples?

File: examples/image-to-text/run_pipeline.py [hunk 1]
```diff
@@ -19,14 +19,19 @@
...
 from transformers import AutoConfig, pipeline
 
 from optimum.habana.transformers.modeling_utils import adapt_transformers_to_gaudi
-
+from optimum.habana.checkpoint_utils import (
+    get_repo_root,
+    model_on_meta,
+    write_checkpoints_json,
+    get_ds_injection_policy,
+)
```
Comment (yafshar) [inline examples/image-to-text/run_pipeline.py:35]:
```suggestion


```

File: examples/image-to-text/run_pipeline.py [hunk 2]
```diff
@@ -149,12 +154,66 @@ def main():
...
-        torch_dtype=model_dtype,
-        device="hpu",
-    )
+    world_size = int(os.getenv("WORLD_SIZE", "1"))
+    rank = int(os.getenv("RANK", "0"))
+
+    if world_size > 1:
+        import deepspeed
+        from habana_frameworks.torch.distributed.hccl import initialize_distributed_hpu
+
```
Comment (yafshar) [inline examples/image-to-text/run_pipeline.py:163]:
```suggestion
        os.environ.setdefault("PT_HPU_ENABLE_LAZY_COLLECTIVES", "true")

```
We can add this as a default for world_size > 1 and using deepspeed, or we should provide a hint in the README for user

**Comment by yuanwu2017** [comment]
> I have tried
> 
> ```shell
> QUANT_CONFIG=./quantization_config/maxabs_measure.json python ../gaudi_spawn.py --world_size 2 --use_deepspeed run_pipeline.py --model_name_or_path llava-hf/llava-v1.6-mistral-7b-hf --image_path "https://llava-vl.github.io/static/images/view.jpg" --use_hpu_graphs --bf16
> ```
> 
> and the command is working on your changes and without it. The same as below,
> 
> ```shell
> python3 ../gaudi_spawn.py --world_size 2 --use_deepspeed  run_pipeline.py \
>     --model_name_or_path llava-hf/llava-v1.6-mistral-7b-hf \
>     --use_hpu_graphs \
>     --bf16
> ```
> 
> Can you comment on the PR?
Yes, it can run without the patch, but it is not in TP mode. That is equivalent to running two separate instances that are not related.

**Comment by yuanwu2017** [comment]
> @yuanwu2017 can you update the README? with information for multi cards run

Ok. I will update later.

**Comment by yafshar** [comment]
@yuanwu2017 would you please address the comment to wrap up this PR. We want to have it in 1.18 release. Thanks

**Comment by yuanwu2017** [comment]
Updated the README. The PR is ok. Please help to review. @regisss @libinta @yafshar

**Comment by libinta** [comment]
@yuanwu2017  it seems we already have multi-card examples now, please check and close the PR.thanks

**Pull request closed without merging**